admiral/cmd/admiral/cmd/root.go

```diff
@@ -254,6 +254,7 @@ func GetRootCmd(args []string) *cobra.Command {
 
 	// Flags pertaining to VS based routing
 	rootCmd.PersistentFlags().BoolVar(&params.EnableVSRouting, "enable_vs_routing", false, "Enable/Disable VS Based Routing")
```

Comment:
This probably needs to be a string with enabled/disabled/all or something

Reply:
@vinay-g would this work? https://github.com/istio-ecosystem/admiral/pull/346#discussion_r1764429203